tibanna_ffcommon/vars.py

```diff
@@ -64,10 +64,13 @@ def BUCKET_NAME(env, filetype):
     global _BUCKET_NAME_LOG
     global _BUCKET_NAME_CWL
 
+    processed_file_types = ['FileProcessed', 'OutputFile']
+    raw_file_types = ['FileFastq', 'FileReference', 'ReferenceFile' 'FileMicroscopy', 'FileSubmitted']
```

Comment:
Are these not also `SubmittedFile`, `ReferenceFile` etc in the same form as `OutputFile` above? Maybe my branch is out of date...

Reply:
Good catch. I added `SubmittedFile`. We haven't done any testing with those, that's why I was missing it.